examples/generated/04.reviewed_preprintv1.json

```diff
@@ -329,4 +337,4 @@
       "previous-step": "_:b2"
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
These changes should be removed or we should regenerate the the examples?

Reply:
Yup